css/hero.css

```diff
@@ -50,4 +50,35 @@ html,body {
   display: flex;
   justify-content: center;
   flex-direction: column;
-}
\ No newline at end of file
+}
+.is-shady {
+animation: flyintoright .4s backwards;
+background: #fff;
+box-shadow: rgba(0, 0, 0, .1) 0 1px 0;
+border-radius: 4px;
+display: inline-block;
+margin: 10px;
+position: relative;
+transition: all .2s ease-in-out;
+}
+.is-shady:hover {
+box-shadow: 0 10px 16px rgba(0, 0, 0, .13), 0 6px 6px rgba(0, 0, 0, .19);
+}
+.is-shady:nth-child(0) {
+animation-delay: .74s
+}
+.is-shady:nth-child(1) {
+animation-delay: .37s
+}
+.is-shady:nth-child(2) {
+animation-delay: .45s
+}
+.is-shady:nth-child(3) {
+animation-delay: .7s
+}
+.is-shady:nth-child(4) {
+animation-delay: .35s
+}
+.is-shady:nth-child(5) {
```

Comment:
This assumes a 5th child, while I appreciate the effort, my goal is to keep the custom css to a minimum to showcase the bulma framework so the templates are easier to customize. Can you remove the shadows?

Reply:
whole point of this pr is in shadows. I dont understand why bulma doesnt have them, and would like to give this ability to templates.

Reply:
@diomed Adding box-shadow is one thing, but you have to remove the animation stuff. A ```.box-shadow``` class with a hover attribute would work, and I have no problem with adding a single class but everything else has to be removed.